src/actions/header.tsx

```diff
@@ -1,7 +1,10 @@
 import * as constants from '../constants/header';
 import { IPerk } from '../types/perks';
-import { IDataRank } from '../types/ranks';
+
```

Comment:
ok, you might want to make this standard on import order cristal clear in documentation.

1. node modules deps
2. interface types
3. services followed by their types

Is that correct ?
Do you have to leave an empty line between 2 and 3 ?